python_dwd/download/download.py

```diff
@@ -45,14 +47,15 @@ def _download_dwd_data(remote_file: Union[str, Path]) -> BytesIO:
         stores data on local file system
 
     """
+    assert isinstance(remote_file, str)
 
     file_server = create_remote_file_name(remote_file)
 
     try:
-        with urllib.request.urlopen(file_server) as url_request:
+        with urlopen(file_server) as url_request:
             zip_file = BytesIO(url_request.read())
-    except urllib.error.URLError as e:
-        raise urllib.error.URLError(f"Error: the stationdata {file_server} couldn't be reached.\n"
+    except URLError as e:
+        raise URLError(f"Error: the stationdata {file_server} couldn't be reached.\n"
```

Comment:
If you raise this error this way, e is an unused variable. You can take first raise `e` and then you specify the error message

Reply:
I removed 'as e'.